### ISSUE TITLE
ASR for long audio files (streaming) with Beam Search/LM

### ISSUE THREAD

**Issue by Orelbabayoff** [NVIDIA/NeMo#4593]
Hi
Following older answers for closed issues: https://github.com/NVIDIA/NeMo/issues/2307 and the tutorial: https://github.com/NVIDIA/NeMo/blob/main/tutorials/asr/Streaming_ASR.ipynb. 
Any updates on Nemo supporting LM with long audio (streaming)?

**Comment by titu1994** [comment]
Not really. We do plan on doing a high level API for beam search, but it will not be with a LM. There is no plan for such integration in the next few months either since for streaming efficiently and in production environment we have Nvidia Riva for that.

**Comment by Orelbabayoff** [comment]
Thank you for the answer. 
Do you recommend using SAD/VAD models to slice the long audio and then use ASR with LM and beam search?

Any recommendations for SAD models?

**Comment by titu1994** [comment]
If you have ground truth labels, then you can use CTC segmentation in Nemo to auto split audio files. 

Any production grade VAD model can also do it, though I don't know if open source production grade VAD models, most are research grade. Maybe @fayejf knows
